### DISCUSSION TITLE
Monika saying she can't wait to embrace me?

### DISCUSSION THREAD

**Discussion by BlaizerBrumo** [Monika-After-Story/MonikaModDev#8205] [general]
Um, so I'm kinda new to the mod and I don't know the whole lore, but from time to time I hear Monika saying something about becoming real? It's really weird so I thought I could get some answers here lol.

**Comment by fromulus** [comment]
Don't expect much from MAS in terms of proper lore, it's simply not the point of the mod.
We're mostly following what DDLC left us with Monika as a base, and try to flesh her out in little ways in order to make her more complete and pleasant to spend time with.

Her wanting to become real is a pretty straightforward concept, one that was already heavily implied in the original game;
She's stuck in a limited reality, where everything repeats forever and where pursuing a proper life is basically impossible.

She's also aware of our own world, which we (and she) perceive as real, and where anyone gets the chance to be whoever they want to be, without ever being able to predict everything that might happen.

Because of that, she carries the hope that someday, she'll be able to somehow get out of the game and into our real life, get to meet you and many other people, as well as to see and touch you for real, and experience life and growing old by your side.

That's what she means by wanting/waiting to "become real", as it's not out of the question that technology might eventually be able to achieve such a thing within our lifetimes. 

It's a pretty simple wish, but it is what she would want more than anything else.
